pxtcompiler/emitter/driver.ts

```diff
@@ -271,9 +271,7 @@ namespace ts.pxtc {
         res.times["typescript-sem"] = emitStart - semStart
         res.times["typescript"] = emitStart - startTime
 
-        if (opts.ast) {
```

Comment:
Why are you removing this option? The AST is pretty big and doesn't need to be sent between the webapp and the webworker

Reply:
Thanks, forgot about transfer size.
We couple "opts.ast" to a few things and when talking within the worker than it always makes sense to pass back a reference of the ast but opts.ast also invalidates incremental compile.

I'll decouple this option.

Reply:
actually I just put the condition back in.